handlers/oauth/oauth_test.go

```diff
@@ -0,0 +1,286 @@
+package oauth
```

Comment:
In the other PR I used the `_test` package pattern. We'll have to decide as a team which we prefer.

Reply:
As in the file names are `*_test.go`? If so then yeah sure; don't see a reason to not do that.

Reply:
You can also have the package be `package oauth_test`, Some reading about this pattern [here](https://github.com/golang/go/issues/25223). I believe it's generally beneficial to do because it causes the testing package to have to import, which can catch some gotchas.